packages/react-core/src/components/Avatar/__tests__/Avatar.test.tsx

```diff
@@ -1,8 +1,20 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import {render, screen } from '@testing-library/react';
 import { Avatar } from '../Avatar';
 
-test('simple avatar', () => {
-  const { asFragment } = render(<Avatar alt="avatar" src="test.png" border="light" />);
-  expect(asFragment()).toMatchSnapshot();
+test('renders simple avatar', () => {
+  render(<Avatar alt="avatar" data-testid="avatar" src="test.png" border="light" />);
+  expect(screen.getByTestId('avatar')).toBeVisible();
 });
+
+test('renders with class name pf-m-light when "light" is passed as border prop', () => {
+  render(<Avatar alt="avatar" data-testid="avatar" src="test.png" border="light" />);
+  expect(screen.getByTestId('avatar')).toHaveClass('pf-m-light');
+});
+
+test('renders with class name pf-m-lg when "lg" is passed as size prop', () => {
+  render(<Avatar alt="avatar" data-testid="avatar" src="test.png" size='lg' />);
+  expect(screen.getByTestId('avatar')).toHaveClass('pf-m-lg');
+});
+
+
```

Comment:
Additional things I would like to see tested:

- That it renders with `pf-c-avatar`
- That the `className` prop functions
- That the `src` prop functions
- That the `alt` prop functions
- That non-explicitly declared props (such as `aria-label` in this case) can be applied as expected
- A snapshot test of the component to prevent unintended changes of the default behavior

Reply:
Also one note: typically we want each test to ideally only include props that are either required or under test, so a lot of the props in use in the tests you have here can be removed.

Reply:
Hopefully all the required changes have been resolved. Thanks for the feedback. I am not quite sure about the way the src test is carried out though, let me know what you think.